k2/python/k2/fsa_properties.py

```diff
@@ -80,3 +80,36 @@ def is_coaccessible(p: int) -> bool:
       False otherwise.
     '''
     return _k2.is_coaccessible(p)
+
+
+def is_epsilon_free(p: int) -> bool:
+    '''Determine whether the given properties imply an FSA 
```

Comment:
imply an FSA that is epsilon free

or

imply an epsilon-free FSA.

Reply:
Fixed